engine/flink/avro-util/src/test/scala/pl/touk/nussknacker/engine/avro/AvroSchemaSpelExpressionSpec.scala

```diff
@@ -171,16 +171,14 @@ class AvroSchemaSpelExpressionSpec extends FunSpec with Matchers {
   }
 
   it("should recognize avro type string as String") {
-    AvroStringSettingsInTests.whenEnabled {
-      val schema = wrapWithRecordSchema(
-        """[
-          |  { "name": "stringField", "type": "string" }
-          |]""".stripMargin)
+    val schema = wrapWithRecordSchema(
+      """[
+        |  { "name": "stringField", "type": "string" }
+        |]""".stripMargin)
```

Comment:
maybe we could expand this tests to also check types MAP and ENUM?
or test it somewhere else?